ForneverMind/posts/en/2021-04-10.net-support-dates.md

```diff
@@ -0,0 +1,77 @@
+| 3.5 SP1                | Yes                    | 2029-01-09          | N/A                     | [Microsoft .NET Framework Lifecycle][lifecycle.net-framework] |
+| 4                      | No                     | 2016-01-12          | N/A                     | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq] |
+| 4.5                    | No                     | 2016-01-12          | N/A                     | [Microsoft .NET Framework Lifecycle][lifecycle.net-framework], [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq] |
+| 4.5.1                  | No                     | 2016-01-12          | N/A                     | [Microsoft .NET Framework Lifecycle][lifecycle.net-framework], [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq] |
+| 4.5.2                  | Yes (extended support) | 2018-10-09          | 2023-10-10              | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows Server 2012 R2 Lifecycle][lifecycle.windows-server-2012-r2] |
+| 4.6                    | Yes                    | 2025-07-29          | N/A                     | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows 10 Enterprise and Education Lifecycle][lifecycle.windows-10.enterprise-edu] |
+| 4.6.1                  | Yes                    | 2025-07-29          | N/A                     | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows 10 Enterprise and Education Lifecycle][lifecycle.windows-10.enterprise-edu] |
+| 4.6.2                  | Yes                    | 2022-01-11          | 2027-01-12              | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows Server 2016 Lifecycle][lifecycle.windows-server-2016] |
+| 4.7                    | Yes                    | 2022-01-11          | 2027-01-12              | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows Server 2016 Lifecycle][lifecycle.windows-server-2016] |
+| 4.7.1                  | Yes                    | 2022-01-11          | 2027-01-12              | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows Server 2016 Lifecycle][lifecycle.windows-server-2016] |
+| 4.7.2                  | Yes                    | 2024-01-09          | 2029-01-09              | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows Server 2019 Lifecycle][lifecycle.windows-server-2019] |
+| 4.8                    | Yes                    | 2024-01-09          | 2029-01-09              | [Lifecycle FAQ - .NET Framework][docs.lifecycle-faq], [Windows Server 2019 Lifecycle][lifecycle.windows-server-2019] |
+
+### Notes
+
+I wasn't able to find any sources for actual .NET Framework 3.5 support dates. [Lifecycle FAQ][docs.lifecycle-faq] sends the reader to the [Lifecycle Policy][lifecycle.net-framework] document, which hasn't .NET Framework 3.5 listed as a separate entry. The [clarification on the support life cycle][support.lifecycle-clarification] lists it as unsupported, though it has no dates:
```

Comment:
"which hasn't .NET listed" doesn't sound like correct English to me. Perhaps "doesn't have .NET listed" is better?